geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/CliFunctionResult.java

```diff
@@ -35,6 +35,7 @@
   private boolean successful;
   private XmlEntity xmlEntity;
   private byte[] byteData = new byte[0];
+  private boolean ignorableFailure = false;
```

Comment:
Can we use a tristate Boolean to indicate this instead of adding another boolean?

Reply:
I agree that a single logical "entry point" for state would be preferable, if we are reasonably sure this class won't show up on the wire during a rolling upgrade.

Maybe an internal `enum StatusStates { SUCCESS, ERROR, IGNORED }`?